be/src/olap/rowset/segment_v2/vertical_segment_writer.cpp

```diff
@@ -138,23 +138,20 @@ void VerticalSegmentWriter::_init_column_meta(ColumnMetaPB* meta, uint32_t colum
     for (uint32_t i = 0; i < column.get_subtype_count(); ++i) {
         _init_column_meta(meta->add_children_columns(), column_id, column.get_sub_column(i));
     }
-    // add sparse column to footer
```

Comment:
Why delete?

Reply:
it's moved to `_append_block_with_variant_subcolumns`